src/devices/BoardLed/samples/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using Iot.Device.BoardLed;
+
+namespace BoardLedSample
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // Open the green led on Raspberry Pi.
+            using BoardLed led = new BoardLed("led0");
+
+            string defaultTrigger = led.Trigger;
+
+            // LED can be controlled only if the trigger is set to none.
+            led.Trigger = "none";
```

Comment:
I'd recommend to use this API for some time and then once we have a better idea on what's useful figure out some better APIs on top of it (i.e. `DisableAllTriggers()`; `RestoreTriggers()`) - I don't expect most of the people to know how to use this unless there is some better option provided. Other option could be something along the lines of:

```csharp
struct Trigger
{
  public static Trigger None => new Trigger("none");

  public string Value { get; private set; }
  public Trigger(string value)
  {
    Value = value;
  }
}
```

and then define couple of other common triggers and leave it as that - otherwise it's really hard to know how this should be used.

Reply:
please ping me if you'll skip it or do it so that I know if I should press the merge button

Reply:
I don't want to change for the time being, because we provide the `EnumerateTriggers()` method for triggers. Or, we can remove the `brightness` property and provide methods to operate the LED. Set the trigger to none inside the method.

```C#
public void Open(int brightness = 255)
{
    Trigger = "none";
    ......
}
public void Close();
```

There's a problem with that. For example, after the user operates the raspberry pi red LED, it will not always light up(restore default trigger after reboot). But if I restore the default trigger in `Dispose()`, then for the user who intends to set the trigger, his settings will fail after the program is finished.